### ISSUE TITLE
raise "ensure => absent" resources

### ISSUE THREAD

**Issue by cjeanneret** [camptocamp/puppet-ghostbuster#36]
Would be great to find the ensure => absent resources in order to be able to clean a bit the catalogs.

**Comment by raphink** [comment]
I don't think this is really dead code. It's code still being parsed and used, but not applying changes in production (hopefully). So I'm in favor of writing a new plugin for that.
